test/chainsaw/testcertificate/cert-recreates/chainsaw-test.yaml

```diff
@@ -0,0 +1,44 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: cert-recreates
+spec:
+  steps:
+  - name: step-00
+    try:
+    - delete:
+        ref:
+          apiVersion: v1
+          kind: Secret
+          namespace: keptn-system
+          name: keptn-certs
+    - script:
+        content: |
+          set -euo pipefail
+          kubectl rollout restart deployment -n keptn-system -l control-plane=lifecycle-operator
+          kubectl rollout restart deployment -n keptn-system -l control-plane=metrics-operator
+    - assert:
+        file: 00-assert.yaml
+  - name: step-01
+    try:
+    - apply:
+        file: 01-delete.yaml
```

Comment:
not sure if to prefer line 10 or this separate file for deleting the secret, but maybe we could be consistent and use the same in both... in the original test we had the first delete embedded in the script to avoid restarting deployments before the delete... probably this is not necessary now...

Reply:
it's not actually doing a delete (not sure it does anything actually), probably generates an empty patch

Reply:
I tracked the issues that came out of this pr in a separate ticket #2955 so no need to solve them here

Reply:
Thanks